lib/oneview-sdk/resource/api300/c7000/enclosure.rb

```diff
@@ -39,22 +39,25 @@ def add_scope(scope)
 
         # Remove one scope from the enclosure
         # @param [OneviewSDK::API300::C7000::Scope] scope The scope resource
+        # @return [Boolean] True if the scope was deleted and false if enclosure has not the scope
         # @raise [OneviewSDK::IncompleteResource] if the uri of scope is not set
         def remove_scope(scope)
           scope.ensure_uri
           scope_index = @data['scopeUris'].find_index { |uri| uri == scope['uri'] }
-          patch('remove', "/scopeUris/#{scope_index}", nil) if scope_index
+          if scope_index
```

Comment:
It would be better to use a guard clause here:

```Ruby
return false unless scope_index
patch('remove', "/scopeUris/#{scope_index}", nil)
true
```